### DISCUSSION TITLE
RunHidden.vbs a good idea?

### DISCUSSION THREAD

**Discussion by FRacine** [PSAppDeployToolkit/PSAppDeployToolkit#997] [General]
As Microsoft is on a way out for vbscript, I am not sure creating a RunHidden.vbs is a good idea. I understand and appreciate someone spent time for this script but I am not sure adding a vbscript is the good way.

**Comment by That-Annoying-Guy** [comment]
I remember one of the authors mentioning that this would be addressed in V4.0

**Reply by FRacine** [reply]
I just opened a case because it is not working on session 0

**Comment by new546-ops** [comment]
No, using RunHidden.vbs to hide scripts is generally not recommended due to potential security risks.

**Comment by mjr4077au** [comment]
This has been addressed in v4.
